subscriber/podaac_data_downloader.py

```diff
@@ -190,6 +190,9 @@ def run(args=None):
             ('GranuleUR[]', cmr_granule),
             ('token', token),
         ]
+        #jmcnelis, 2023/06/14 - provide for wildcards in granuleur-based search
+        if '*' or '?' in cmr_granule:
```

Comment:
@jjmcnelis I think this statement will cause this if statement to be entered in every case. 

```python3
>>> cmr_granule = 'SWOTCalVal_??_GNSS_L2_*'
>>> '*' or '?' in cmr_granule
'*'
>>> cmr_granule = 'SWOTCalVal_GNSS_L2_'
>>> '*' or '?' in cmr_granule
'*'
```

The logic should be

```suggestion
        if '*' in cmr_granule or '?' in cmr_granule:
```

```python3
>>> cmr_granule = 'SWOTCalVal_??_GNSS_L2_*'
>>> '*' in cmr_granule or '?' in cmr_granule
True
>>> cmr_granule = 'SWOTCalVal_GNSS_L2_'
>>> '*' in cmr_granule or '?' in cmr_granule
False
```

Reply:
Oof, that's embarrassing. ;) Thanks @skorper. (I didn't know that what I put is even valid python syntax..)Will make this fix and resubmit.

Reply:
I did??